pkgs/development/interpreters/python/update-python-libraries/update-python-libraries.py

```diff
@@ -344,12 +411,18 @@ def _update_package(path, target):
     # sri hashes have been the default format since nix 2.4+
     sri_hash = subprocess.check_output(["nix", "--extra-experimental-features", "nix-command", "hash", "to-sri", "--type", "sha256", new_sha256]).decode('utf-8').strip()
```

Comment:
Further up (~282) we call the same command. Factor out?

Reply:
Extracted into `_hash_to_sri` function.